### DISCUSSION TITLE
⛔ 🛟 Support Request: Slow Response to Issues and Feature Requests in Elementor 🚨

### DISCUSSION THREAD

**Discussion by ayoubkhan558** [elementor/elementor#22945] [Developers Community]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The Elementor team seems to be slow in responding to issues and implementing new features requested by developers. This has been a recurring concern, with developers waiting for years for the addition of necessary features, while unnecessary ones like AI for Image Creation are being added. This lack of responsiveness is hindering the growth and potential of Elementor.
![image](https://github.com/elementor/elementor/assets/99708427/afbdfecd-0ad2-4038-8ddf-fc6dab88d87f)

### Describe the solution you'd like

The Elementor team should prioritize listening to developers' feedback and implementing their feature requests in a timely manner. 
❌
For instance, a feature request (#4246) was created on Apr 25, 2018, to configure conditions to show or hide blocks or sections. It's been five years and this issue is still unresolved. This feature is a basic functionality that other page builders like Breakdance have added by default within a year of their launch.

### Describe alternatives you've considered

If the Elementor team continues to disregard developers' feedback, developers may consider using other page builders like Oxygen, Bricks, or Breakdance, which have shown a consistent track record of listening to developers and implementing their feature requests promptly. ☠️⚠️

### Additional context

Developers' time is limited and precious. 
😡🥵
The lack of responsiveness from the Elementor team is not only disappointing but also discouraging for developers who are considering whether to continue using Elementor or switch to other page builders. The Elementor team needs to understand the importance of developers' feedback and act on it promptly to maintain its user base and stay competitive in the market.

**Comment by pixelloop** [comment]
I couldn't agree more. Elementor really needs to sort their priorities out, and instead of investing massive resources in new non-essential features, they should focus their attention on adding basic features developers need and expect. Many of which have been built into other page builders for years. 

For what it's worth, Elementor is a fun page builder to work with but it's also the most frustrating when you need something basic and can't understand how it hasn't been added yet.

**Comment by 79ho3ein** [comment]
I started with elementor 5 years ago and really enjoyed it. Currently, I only work with Elementor. But to be honest, I'm thinking about leaving Elementor these days. Because new and great features are released, but after a few updates, they are still not usable. Like css grid and mega manu.
Before releasing new features, it is important that reported bugs are identified and fixed quickly. But unfortunately, we see that Megamanu still has basic bugs that have been abandoned after 2 versions.
I hope the Elementor team will reach a basic conclusion to overcome this situation...

**Comment by lensbreak** [comment]
Everyone here saying the same things as on my viral post in the official group:
https://www.facebook.com/groups/Elementors/?multi_permalinks=6078009698991432

**Comment by ayoubkhan558** [comment]
Here all time top discussion to add new features, but elementor team isn't interested at all. 

You can check there website for roadmap, there is no plan to add these in upcoming releases. 

😡💢 😡

![Screenshot_2023-07-03-21-02-48-83_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/elementor/elementor/assets/99708427/f5bf83a0-37bc-4c6a-8829-9f1005c90ba1)
![Screenshot_2023-07-03-21-02-41-20_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/elementor/elementor/assets/99708427/7fa6686c-f1fa-49ba-8b16-7095da1fc7d8)

**Comment by ghost** [comment]
For a little/simple functionality we have to add third party plugin, which often comes with other widgets, add more JavaScript, css assets at the end it effect performance. 
Please Elementor focus on these simple, but much need functionalities, so we don't have to depend on third party plugins.

**Comment by nicholaszein** [comment]
Hello

Thanks for contacting us! 🙏 

### 🛟 This is more of a support request, not something we can help with here. 

Please refer to our [guidelines](https://github.com/orgs/elementor/discussions/23196) on how to submit Feature Requests. This is not the place to submit complaints. Thank you for understanding.


### ⚠️ Note that GitHub discussions should not be used for support.

❔ If you have complaints, questions, or need guidance, help, or general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) The main Facebook group where all kinds of users come together to help each other.
- [Elementor Academy](https://elementor.com/academy) Learn everything you need to know about using Elementor.
- [Elementor Developers Documentation](https://developers.elementor.com/) Learn everything about our Developer-API.
- [Free Support Forum](https://wordpress.org/support/plugin/elementor) Get help on WordPress.org.

💎 If you have an active [Elementor Pro](https://elementor.com/pro) license you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

Kind regards
